tests/python/frontend/pytorch_neo/test_forward.py

```diff
@@ -0,0 +1,608 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=import-self, invalid-name, unused-argument
+"""Unit tests for various models and operators"""
+from time import time
+import os
+import sys
+from tempfile import TemporaryDirectory
+from scipy.stats import t as tdistr
+import numpy as np
+import torch
+import tvm
+import torchvision
+import single_op
+
+
+from tvm import relay
+from tvm.contrib import graph_runtime
+#from tvm.relay.testing.config import ctx_list
+
+sys.setrecursionlimit(10000)
+
+TARGET = 'llvm'
+CTX = tvm.cpu()
```

Comment:
why we only test for cpu

Reply:
Was an artifact from the NNVM parser. Removed after switching to use ctx_list.